test/azul_test_case.py

```diff
@@ -52,6 +56,10 @@ class AzulTestCase(TestCase):
     get_credentials_boto3 = None
     _saved_boto3_default_session = None
 
+    # Region corresponds to south asia (Mumbai). Must be a real region because
+    # SQS will verify this.
+    _aws_region_mock = patch.dict(os.environ, AWS_DEFAULT_REGION='ap-south-1')
```

Comment:
```suggestion
    _aws_region_mock = patch.dict(os.environ, AWS_DEFAULT_REGION='us-gov-west-1')
```

we are pretty much guaranteed to not have access to that region.

Reply:
Moto/boto does not consider this to be a valid region in certain contexts:

```
======================================================================
ERROR: test_versioning (service.test_version_service.TestVersionService)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/DataBiosphere/azul/test/service/test_version_service.py", line 19, in setUp
    super().setUp()
  File "/home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/moto/core/models.py", line 80, in wrapper
    result = func(*args, **kwargs)
  File "/home/travis/build/DataBiosphere/azul/test/version_table_test_case.py", line 23, in setUp
    dict(AttributeName=VersionService.key_name, KeyType='HASH'),
  File "/home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/botocore/client.py", line 357, in _api_call
    return self._make_api_call(operation_name, kwargs)
  File "/home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/botocore/client.py", line 648, in _make_api_call
    operation_model, request_dict, request_context)
  File "/home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/botocore/client.py", line 667, in _make_request
    return self._endpoint.make_request(operation_model, request_dict)
  File "/home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/botocore/endpoint.py", line 102, in make_request
    return self._send_request(request_dict, operation_model)
  File "/home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/botocore/endpoint.py", line 137, in _send_request
    success_response, exception):
  File "/home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/botocore/endpoint.py", line 231, in _needs_retry
    caught_exception=caught_exception, request_dict=request_dict)
  File "/home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/botocore/hooks.py", line 356, in emit
    return self._emitter.emit(aliased_event_name, **kwargs)
  File "/home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/botocore/hooks.py", line 228, in emit
    return self._emit(event_name, kwargs)
  File "/home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/botocore/hooks.py", line 211, in _emit
    response = handler(**kwargs)
  File "/home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/botocore/retryhandler.py", line 183, in __call__
    if self._checker(attempts, response, caught_exception):
  File "/home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/botocore/retryhandler.py", line 251, in __call__
    caught_exception)
  File "/home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/botocore/retryhandler.py", line 269, in _should_retry
    return self._checker(attempt_number, response, caught_exception)
  File "/home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/botocore/retryhandler.py", line 317, in __call__
    caught_exception)
  File "/home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/botocore/retryhandler.py", line 223, in __call__
    attempt_number, caught_exception)
  File "/home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/botocore/retryhandler.py", line 359, in _check_caught_exception
    raise caught_exception
  File "/home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/botocore/endpoint.py", line 197, in _do_get_response
    responses = self._event_emitter.emit(event_name, request=request)
  File "/home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/botocore/hooks.py", line 356, in emit
    return self._emitter.emit(aliased_event_name, **kwargs)
  File "/home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/botocore/hooks.py", line 228, in emit
    return self._emit(event_name, kwargs)
  File "/home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/botocore/hooks.py", line 211, in _emit
    response = handler(**kwargs)
  File "/home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/moto/core/models.py", line 298, in __call__
    status, headers, body = response_callback(request, request.url, request.headers)
  File "/home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/moto/core/responses.py", line 117, in dispatch
    return cls()._dispatch(*args, **kwargs)
  File "/home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/moto/core/responses.py", line 207, in _dispatch
    return self.call_action()
  File "/home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/moto/core/utils.py", line 264, in _wrapper
    response = f(*args, **kwargs)
  File "/home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/moto/dynamodb2/responses.py", line 65, in call_action
    response = getattr(self, endpoint)()
  File "/home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/moto/dynamodb2/responses.py", line 119, in create_table
    table = self.dynamodb_backend.create_table(table_name,
  File "/home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/moto/dynamodb2/responses.py", line 57, in dynamodb_backend
    return dynamodb_backends[self.region]
KeyError: 'us-gov-west-1'
----------------------------------------------------------------------
Ran 322 tests in 817.926s
```

Reply:
I believe the list of regions that will not trigger this error can be found in `.venv/lib/python3.6/site-packages/botocore/data/endpoints.json`

Reply:
```
$ cat .venv/lib/python3.6/site-packages/botocore/data/endpoints.json | jq .partitions[].regions | jq keys[]
"ap-east-1"
"ap-northeast-1"
"ap-northeast-2"
"ap-south-1"
"ap-southeast-1"
"ap-southeast-2"
"ca-central-1"
"eu-central-1"
"eu-north-1"
"eu-west-1"
"eu-west-2"
"eu-west-3"
"me-south-1"
"sa-east-1"
"us-east-1"
"us-east-2"
"us-west-1"
"us-west-2"
"cn-north-1"
"cn-northwest-1"
"us-gov-east-1"
"us-gov-west-1"
"us-iso-east-1"
"us-isob-east-1"
```

Reply:
More directly and correctly:
```
>>> import moto.dynamodb2.models as m
>>> m.dynamodb_backends.keys()
dict_keys(['ap-east-1', 'ap-northeast-1', 'ap-northeast-2', 'ap-south-1', 'ap-southeast-1', 'ap-southeast-2', 'ca-central-1', 'eu-central-1', 'eu-north-1', 'eu-west-1', 'eu-west-2', 'eu-west-3', 'me-south-1', 'sa-east-1', 'us-east-1', 'us-east-2', 'us-west-1', 'us-west-2'])
```